conf/eslint-recommended.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
         "no-compare-neg-zero": "error",
         "no-cond-assign": "error",
         "no-const-assign": "error",
+        "no-constant-binary-operand": "error",
```

Comment:
Rules shouldn't be added as `recommended`. We can only add to `recommended` in a major release.